analytics/usage/src/main/kotlin/com/malinskiy/marathon/usageanalytics/tracker/GrafanaCloud.kt

```diff
@@ -0,0 +1,71 @@
+package com.malinskiy.marathon.usageanalytics.tracker
+
+import com.malinskiy.marathon.usageanalytics.Event
+import okhttp3.MediaType.Companion.toMediaType
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import okhttp3.RequestBody.Companion.toRequestBody
+import java.lang.Exception
+import java.util.UUID
+
+class GrafanaCloud : UsageTracker {
+    private val client = OkHttpClient()
+    private val key =
```

Comment:
This key will end up in our sources anyways, don't see any reason to make it harder to extract